server/src/main/scala/ch.epfl.scala.index.server/Server.scala

```diff
@@ -60,7 +62,13 @@ object Server {
         dependencies <- api.dependencies()
         latestProjects <- api.latestProjects()
         latestReleases <- api.latestReleases()
-      } yield views.html.frontpage(keywords, targets, dependencies, latestProjects, latestReleases, userInfo)
```

Comment:
hold on let's merge not formating PR first since they will be resolved automatically then merge the formatting rebased to master